src/course/en/articles.json

```
@@ -37,24 +39,38 @@
             "_banks": {
                 "_isEnabled": true,
                 "_split": "2,2",
-                "_randomisation": true
+                "_randomisation": false
```

please revert back to old settings
```suggestion
                "_randomisation": true
```